nixos/doc/manual/installation/installing.chapter.md

```diff
@@ -1,30 +1,147 @@
+     and hold the power button.
+     :::
+
+     ::: {.note}
+     If your computer support both BIOS and UEFI boot, choose the UEFI option.
+     :::
+
+     ::: {.note}
+     If you use a CD for the installation, the computer will probably boot from
+     it automatically. If not, choose the option containing the word "CD" from
+     the boot menu.
+     :::
+
+3.   Shortly after selecting the appropriate boot drive, you should be
+     presented with a menu with different installer options. Leave the default
+     and wait (or press <kbd>Enter</kbd> to speed up).
+
+4.   The graphical images will start their corresponding desktop environment
+     and the graphical installer, which can take some time. The minimal images
+     will boot to a command line. You have to follow the instructions in
+     [](#sec-installation-manual) there.
+
+## Graphical Installation {#sec-installation-graphical}
+
+The graphical installer is recommended for desktop users and will guide you
+through the installation.
+
+1.   In the "Welcome" screen, you can select the language of the Installer and
+     the installed system.
+
+     ::: {.tip}
+     Leaving the language as "American English" will help to be able to copy
+     and paste error messages in a search engine or issue report.
+     :::
+
+2.   Next you should choose your location to have the timezone set correctly.
+     You can actually click on the map!
+
+     ::: {.note}
+     The installer will use an online service to guess your location based on
+     your public IP address.
+     :::
+
+3.   Then you can select the keyboard layout. The default keyboard model should
+     work well with most desktop keyboards. If you have a special keyboard or
+     notebook, your model might be in the list. Select the language you are most
+     comfortable typing in.
+
+     ::: {.tip}
+     There are special keyboard layouts that are optimized to be ergonomic like
+     [NEO](https://www.neo-layout.org/),
+     [Dvorak](https://en.wikipedia.org/wiki/Dvorak_keyboard_layout) and
+     [Colemak](https://colemak.com/), in opposition to the classic QWERTY layout
+     which comes from typewriters and has not changed much since 1873.
+     You might want to check them out.
+     :::
+
+4.   On the "Users" screen, you have to type in your name, login name and
+     password. You can also enable an option to automatically login to the
+     desktop.
+
+5.   Then you have the option to choose a desktop environment. If you want to
+     create a custom setup with a window manager, you can select "No desktop".
+
+     ::: {.tip}
+     If you don't have a favorite desktop and don't know which one to choose,
+     you can stick to either GNOME or Plasma. They have a quite different
+     design, so you should be able to choose which one you like better.
+     They are both popular choices and well tested on NixOS.
+     :::
+
+6.   You have the option to allow unfree software in the next screen.
```

Comment:
Does this include unfree firmware? If not we should likely add a short section for that, too.

Reply:
yes. there is a detailed description in the installer

![Screenshot from 2022-05-30 02-51-00](https://user-images.githubusercontent.com/91113/170899056-3bd32229-4048-4ac1-b471-4ae340efe2c9.png)


(it does not enable unfree gpu drivers, but e.g. wifi drivers)